Projects/Career-Crafters/app.js

```diff
@@ -10,6 +10,7 @@ $(".menu-bar").on("click",function(){
     $(".sidebar").addClass("active")
 });
 
-$(".logo").on("click",function(){
+$(".btn").on("click",function(){

```

Comment:
![image](https://github.com/apu52/METAVERSE/assets/114172928/e68e0f6e-30ce-420f-b9ea-cc8d281438c2)

it is not working .Please try to recheck the functions u have add

Reply:
That back button will only work if the hamburger is required to open the sidebar or else it will be static.